lib/components/SCardHeader.vue

```diff
@@ -8,15 +8,23 @@
       <template v-if="actions.length > 0">
         <template v-for="(action, index) in actions">
           <SToolTip :key="index" :text="action.disabled">
-            <component
-              :is="action.link ? 'router-link' : 'button'"
+            <RouterLink
+              v-if="action.link"
               class="action"
               :class="[action.mode || 'neutral', { disabled: action.disabled }]"
               :to="action.link"
-              @click="action.callback && !action.disabled ? action.callback() : () => {}"
+              event
             >
               <component :is="getIcon(action.icon, !!action.disabled)" class="action-icon" />
-            </component>
+            </RouterLink>
+            <button
+              v-else
+              class="action"
+              :class="[action.mode || 'neutral', { disabled: action.disabled }]"
+              @click="action.callback()"
```

Comment:
We should stop calling the callback if `disabled: true`?

```suggestion
              @click="action.callback && !action.disabled ? action.callback() : () => {}"
```